akka-cluster/src/main/scala/akka/cluster/Member.scala

```diff
@@ -129,6 +129,7 @@ object Member {
    * INTERNAL API
    * Orders the members by their address except that members with status
    * Joining, Exiting and Down are ordered last (in that order).
+   * FIXME include shutting down
```

Comment:
^ ?

Reply:
I don't think this needs to be added but was just making sure i thought it through, can be a leader like any Up member